modules/openxr/extensions/openxr_extension_wrapper_extension.cpp

```diff
@@ -287,8 +287,7 @@ void OpenXRExtensionWrapperExtension::register_extension_wrapper() {
 	OpenXRAPI::register_extension_wrapper(this);
 }
 
-OpenXRExtensionWrapperExtension::OpenXRExtensionWrapperExtension() :
-		Object(), OpenXRExtensionWrapper() {
+OpenXRExtensionWrapperExtension::OpenXRExtensionWrapperExtension() {
```

Comment:
This is the only bit I'm iffy on, as I'm not positive if these constructors were explicit for a reason. They *should* be able to setup implicitly, but there could be something about multi-inheritance classes I'm not aware of.

Reply:
@BastiaanOlij @m4gr3d If you wanted to take a look.

Reply:
It will probably not be a problem, I'd have to test, however on principle I don't like this change and prefer explicit notation over implicit.

The problem you introduce, especially when a large refactor is needed when someone adds new constructors to base classes, is that you introduce a ton of guess work on the compilers part and thus a ton of work on the developer.

When being explicit, it might be a little extra code, but as a developer you are communicating to a maintainer that looks at your code weeks/months/years from now, exactly what the intention was, even if the base class may have changed since.

Reply:
(though with the footnote that this imho would only apply when any parameters are in play, so this scenario may still be considered overkill)

Reply:
While I generally prefer explicit notation as well, `readability-redundant-member-init` only applies when the parameters would match the default values. I can absolutely see the value in disabling it for niche cases where information about the arguments should be explicitly conveyed (ex: default values aren't necessarily zeroed-out, a constructor performs some non-trivial action, etc). For most cases though, parameters will only need to be passed if a default value is being changed.

Reply:
I think the change is good, this is the odd one out throughout the whole codebase, and it gives the impression of multi-inheritance (which is a no go for Godot) when actually `OpenXRExtensionWrapperExtension` already inherits `Object`, so the explicit `Object` constructor is doubly redundant.